pom.xml

```diff
@@ -0,0 +1,90 @@
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>uk.gov.companieshouse</groupId>
+    <artifactId>web.accounts</artifactId>
+    <version>0.1.0</version>
+    <packaging>jar</packaging>
+
+    <name>web.accounts</name>
+    <description>Web service to handle company account data</description>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.0.3.RELEASE</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <java.version>1.8</java.version>
+
+        <lombok.version>1.18.0</lombok.version>
+        <thymeleaf-extras.version>3.0.1.RELEASE</thymeleaf-extras.version>
+        <structured-logging.version>1.2.0-rc1</structured-logging.version>
+    </properties>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-thymeleaf</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.thymeleaf.extras</groupId>
+            <artifactId>thymeleaf-extras-java8time</artifactId>
+            <version>${thymeleaf-extras.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>uk.gov.companieshouse</groupId>
+            <artifactId>structured-logging</artifactId>
+            <version>${structured-logging.version}</version>
+            <scope>compile</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${lombok.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+                <configuration>
+                    <mainClass>uk.gov.companieshouse.web.accounts.Application</mainClass>
+                    <layout>ZIP</layout>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>repackage</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+
```

Comment:
I don't see any resolution repositories? How is someone supposed to build the project if you haven't specified the artifactory in which the dependencies are located?

Reply:
This is the barebones pom, and doesn't contain everything needed. The repository also doesn't contain all the needed dependencies for the repositories, doesn't contain a `Makefile`, nor a start script.

Reply:
My comment referenced the artifactories because you had structured logging as a dependency but as it has been removed my first comment is redundant as all dependencies are at mvncentral.